recipe/meta.yaml

```diff
@@ -1,73 +1,75 @@
-{% set version = "0.4.1" %}
+{% set name = "dioptas" %}
+{% set version = "0.5.7" %}
 
 package:
-  name: dioptas
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/Dioptas/Dioptas/archive/{{ version }}.tar.gz
-  sha256: d2cf86417429f5092fc9db7d7a745706c027dafac1ee546c9a87eb5fd6d0d335
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dioptas-{{ version }}.tar.gz
+  sha256: cb3019da250aada747ed024d7ed473ddc23be10b22aa996fd076656d5045ca23
 
 build:
-  number: 1002
-  skip: true  # [win and py27]
-  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  skip: true  # [py<39]
```

Comment:
[Release note](https://github.com/Dioptas/Dioptas/releases/tag/0.5.7) states "dropping support for python 3.6, 3.7 and 3.8", let's not package Dioptas for python3.8